channels/ad.m3u

```diff
@@ -1,5 +1,3 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Catalan" tvg-logo="https://i.imgur.com/kJCjeQ4.png" group-title="",RTVA
-http://videos.rtva.ad:1935/live/rtva/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Catalan" tvg-logo="https://i.imgur.com/kJCjeQ4.png" group-title="",RTVA
-http://videos.rtva.ad:1935/live/web/playlist.m3u8
```

Comment:
I would consider keeping one of these as a backup.

Reply:
Noted.